enterprise_access/apps/api/v1/tests/test_subsidy_access_policy_views.py

```diff
@@ -1705,6 +1705,7 @@ def test_credits_available_endpoint_with_content_assignments(
             'remaining_balance': 5000,
             'subsidy_expiration_date': '2030-01-01 12:00:00Z',
             'learner_content_assignments': [expected_learner_content_assignment],
+            'learner_requests': [],
```

Comment:
[inform] Fixes an existing, unrelated failing test that was committed to `main`...